v1.0/docs/api-reference/definitions.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
I show those files.  I will fix the mess right now.  In general the  v1.0/docs and v1.0/exampes directories are slaved to the release-1.0 tree.  you MUST NOT add files in gh-pages that are not in release-1.0